UnleashClient/__init__.py

```diff
@@ -185,7 +184,32 @@ def __init__(
 
         self.metrics_headers: dict = {}
 
-        # Scheduler bootstrapping
+        self._init_scheduler(scheduler, scheduler_executor)
+
+        if custom_strategies:
+            self.engine.register_custom_strategies(custom_strategies)
+
+        self.strategy_mapping = {**custom_strategies}
+
+        # Client status
+        self.is_initialized = False
+
+        # Bootstrapping
+        if self.unleash_bootstrapped:
+            OfflineConnector(
+                engine=self.engine,
+                cache=self.cache,
+                scheduler=None,
+            ).load_features()
+
+        self.connector: Union[OfflineConnector, PollingConnector] = None
```

Comment:
Absolute nit but can we move this up to where all the other fields on this class are declared?

Reply:
_init_scheduler? yes, but it triggers 'too many statements' linting error